scripts/install-deps.sh

```diff
@@ -201,8 +204,8 @@ install_deps_FreeBSD()
         qt5-qmake \
         qt5-widgets \
         range-v3 \
+        yaml-cpp
```

Comment:
Oh you have gotten that to work? That would be awesome. I prefer system deps whenever possible.